platform/default/sqlite3.hpp

```diff
@@ -42,27 +48,23 @@ class Database {
 
     explicit operator bool() const;
 
-    static void errorLogCallback(void *arg, const int err, const char *msg);
     void setBusyTimeout(std::chrono::milliseconds);
     void exec(const std::string &sql);
     Statement prepare(const char *query);
 
-    int64_t lastInsertRowid() const;
```

Comment:
> This sounds like the way to go

Agreed. My hunch is that this is what Qt does internally for its own SQLite implementation. Can you verify that? If so, we only need to do it ourselves for the non-Qt implementation.